src/server/migrations/20200510015322_create_modules_schema.js

```diff
@@ -1,24 +1,12 @@
 // TODO: This is a sample migration for demonstration. Remove before adding real ones.
 
 exports.up = function(knex) {
-  return knex.schema.createTable('modules', (table) => {
+  return knex.schema.createTable('Topics', (table) => {
```

Comment:
Please use lowercase name `topics` for consistency.

Reply:
updated toni